libminifi/src/c2/C2Agent.cpp

```diff
@@ -187,6 +209,22 @@ void C2Agent::configure(const std::shared_ptr<Configure> &configure, bool reconf
     }
   }
 
+  std::string trigger_classes;
+  if (configure->get("nifi.c2.agent.trigger.classes", "c2.agent.trigger.classes", trigger_classes)) {
```

Comment:
Looks like this is missing from the docs to specify a class or should be assumed when the property pertaining to the file update trigger is provided in the configuration.